### ISSUE TITLE
Adopt BIDS derivative standard for disc labels

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/manual-correction#87]
When running `manual_correction.py` following the [wiki tutorial](https://github.com/spinalcordtoolbox/manual-correction/wiki#vertebral-labeling), the created labeled had the suffix `_label-disc`.

It should instead be: `_label-discs_dlabel` as per our [internal doc](https://intranet.neuro.polymtl.ca/data/dataset-curation.html#examples-and-use-cases)

**Comment by valosekj** [comment]
Good point! Indeed, the code has not yet been updated according to the latest information.

Related issue: https://github.com/spinalcordtoolbox/manual-correction/issues/77

Assigning @NathanMolinier
